framework/TSA/Factory.py

```diff
@@ -21,9 +21,12 @@
 from .Fourier import Fourier
 from .ARMA import ARMA
 from .Wavelet import Wavelet
+from .AutoCorrelation import AutoCorrelation
+from .PolynomialRegression import PolynomialRegression
 
 factory = EntityFactory('TimeSeriesAnalyzer')
 # TODO map lower case to upper case, because of silly ROM namespace problems
 aliases = {'Fourier': 'fourier',
-           'ARMA': 'arma'}
+           'ARMA': 'arma',
+           'Wavelet': 'wavelet'}
```

Comment:
what about ACF?

Reply:
Interestingly autocorrelation does not need to be mapped like Wavelet or Arma,not sure if this is an issue or not.